libmuscle/python/libmuscle/manager/manager.py

```diff
@@ -90,13 +90,18 @@ def get_server_location(self) -> str:
     def start_instances(self) -> None:
         """Starts all required component instances."""
         if self._run_dir is None:
-            raise RuntimeError('No run dir specified')
+            message = 'No run dir specified'
+            _logger.error(message)
+            raise RuntimeError(message)
         if not self._instance_manager:
-            raise RuntimeError(
+            message = (
                     'For MUSCLE3 to be able to start instances, the'
                     ' configuration must contain a model, implementations,'
                     ' and resources. Please make sure they are all there.')
+            _logger.error(message)
+            raise RuntimeError(message)
         try:
+            self._configuration.as_configuration().check_consistent()
```

Comment:
Why would we do this here? Checking the configuration doesn't have anything to do with starting the instances. Sure, we need a valid configuration, but we're already doing some checks in `muscle_manager.py`, so why not call `check_consistent()` there as well?